lang/cs/Org.Apache.REEF.Driver/Bridge/ClrHandlerHelper.cs

```diff
@@ -35,18 +34,41 @@ namespace Org.Apache.REEF.Driver.Bridge
 {
     public class ClrHandlerHelper
     {
+        private const string DllExtension = ".dll";
         private static readonly Logger LOGGER = Logger.GetLogger(typeof(ClrHandlerHelper));
 
+        /// <summary>
+        /// The set of REEF assemblies required for the Driver.
+        /// </summary>
         public static string[] ReefAssemblies
         {
             get
             {
-                return new[] { "Microsoft.Hadoop.Avro.dll", "Org.Apache.REEF.Driver.dll", "Org.Apache.REEF.Common.dll", "Org.Apache.REEF.Utilities.dll", "Org.Apache.REEF.Network.dll", "Org.Apache.REEF.Tang.dll", "Org.Apache.REEF.Wake.dll", "Newtonsoft.Json.dll", "protobuf-net.dll" };
+                return new[]
+                {
+                    "Microsoft.Hadoop.Avro.dll", 
+                    "Org.Apache.REEF.Driver.dll", 
+                    "Org.Apache.REEF.Common.dll", 
+                    "Org.Apache.REEF.Utilities.dll", 
+                    "Org.Apache.REEF.Network.dll", 
+                    "Org.Apache.REEF.Tang.dll", 
+                    "Org.Apache.REEF.Wake.dll", 
+                    "Newtonsoft.Json.dll", 
+                    "protobuf-net.dll"
+                };
             }
         }
 
+        /// <summary>
+        /// The memory granularity in megabytes for the Evaluator Descriptors.
+        /// </summary>
         internal static int MemoryGranularity { get; set; }
```

Comment:
This strikes me as a code smell. `static` mutable fields in a multithreaded program are an easy way to create problems. Is this used? Can we stop using it?